test/test_jit.py

```diff
@@ -1568,6 +1571,27 @@ def test_quant_fusion(self):
             torch._C._jit_pass_quant_fusion(graph)
             FileCheck().run(input_str, graph)
 
+    def test_quantized_conv_relu_fusion(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.conv = torch.nn.Conv2d(1, 4, 2, 3).float()
+                self.relu = torch.nn.ReLU()
+
+            def forward(self, x):
+                return self.relu(self.conv(x))
+
+        m = torch.jit.script(M().eval())
+        m = prepare_script(m, {'': script_qconfig(default_qconfig)}, True)
+        data = torch.randn(1, 1, 10, 10, dtype=torch.float)
+        m(data)
+        m = convert_script(m, True)
+        print(m.graph_for(data))
+        FileCheck().check_not("aten::conv2d") \
+                   .check_not("aten::relu") \
+                   .check("quantized::conv2d_relu") \
+                   .run(m.graph_for(data))
```

Comment:
Are there other checks that one can do: i.e check if there is a single inlined graph and that all weight tensors are constants?

Reply:
you mean the functionality of freeze? doesn't feel like that's something we should be testing here